circle.yml

```diff
@@ -6,6 +6,7 @@ machine:
     PARALLEL_TEST_PROCESSORS: 8
 
     PATH: $HOME/.cabal/bin/:$PATH
+    GO_PROJECT: "$HOME/.go_workspace/src/github.com/Originate/gt"
```

Comment:
You should make this `$HOME/.go_workspace/src/github.com/Originate/git-town` otherwise if you ever have sub-packages, go will get confused